frontend/components/product-list/sections/HeroSection.tsx

```diff
@@ -34,9 +34,9 @@ export function HeroSection({ productList }: HeroSectionProps) {
             {page > 1 ? ` - Page ${page}` : ''}
          </HeroTitle>
          {productList.tagline && productList.tagline.length > 0 && page === 1 && (
-            <Text fontWeight="bold" fontSize="xl" px={{ base: 6, sm: 0 }}>
+            <h2>
                {productList.tagline}
-            </Text>
+            </h2>
```

Comment:
This changes the look & feel of the tagline

| Before  | With this PR |
| ------------- | ------------- |
| <img width="997" alt="Screenshot 2022-06-28 at 07 43 37" src="https://user-images.githubusercontent.com/4640135/176102172-b5fca44a-f3fd-4fe5-b9b1-59815976d28c.png">  | <img width="1123" alt="Screenshot 2022-06-28 at 07 44 03" src="https://user-images.githubusercontent.com/4640135/176102245-f7741729-5772-4eec-8089-6e82e46d9105.png">  |

Reply:
Hint: You can use `as` prop also for the Text component, to change its semantics

Reply:
Resolved here cc056d290ff4ad93fba17296101b2bce33bc60f9